### ISSUE TITLE
Can't seem to start daemonless windows

### ISSUE THREAD

**Issue by Tcll** [linuxmint/nemo#3074]
```
 * Nemo version 4.4.2
 * Is issue with desktop or windowed nemo? (custom AppImage)
 * Distribution - Artix OpenRC
 * Graphics hardware *and* driver used AMDGPU
 * 64 bit
 ```

**Issue**
all windows close when daemon closes or crashes

**Steps to reproduce**
open 2 separate terminals and start nemo in both
in the first terminal press ` Ctrl `+` C ` to stop the daemon

**Expected behaviour**
each terminal should start it's own daemon
and only the first window should close when ` Ctrl `+` C ` is pressed in the first terminal

**Other information**
apparently `sudo -u <user>` actually works for a friend to start individual daemons as wanted
however it doesn't work for me and I'm still stuck on a single daemon regardless (same AppImage as well)

why do I want to run individual daemons?
it's quite common that Nemo will crash after extended pariods
at this moments I'll end up losing more than 30 windows with more than 200 tabs in total across 16 workspaces
running more daemons should reduce the losses caused by crashing daemons

**Comment by Tcll** [comment]
update
looks like the problem has to do with a dbus check in:
- `libnemo-private/nemo-dbus-manager.c` for `"/org/Nemo"`
- `src/nemo-freedesktop-dbus.c` for `"/org/freedesktop/FileManager1"`

**Comment by Tcll** [comment]
sorry I can't edit without enabling malware
but I can get around the issue with `firejail --nodbus ./AppRun`

but of course that breaks everything else regarding dbus like Open With integrations

**Comment by TopherIsSwell** [comment]
I say this as someone who knows nothing about Nemo or dbus, but it appears that this behavior is a result of hard-coding the dbus service name of "/org/Nemo". @NemoDevelopers -- would it be tenable to make the dbus service name overrideable at the command-line to allow multiple instances to coexist simultaneously?

@Tcll , Have you uploaded logs or a coredump or something after Nemo crashes and see if the problem could get fix from that angle?

**Comment by Tcll** [comment]
> would it be tenable to make the dbus service name overrideable at the command-line to allow multiple instances to coexist simultaneously?

yeah, this is actually what I was hoping to achieve out of this ever since the second message :)

that said though, that would only apply if DBus is actually used
additionally I'd like to make a few nudges (to those involved) towards efforts to avoid relying on DBus
perhaps making it more viable to use plugin/script functionality for integrations

after messing around with DBus, lets just say I'd like to completely remove it from my system :)

> Have you uploaded logs or a coredump or something after Nemo crashes and see if the problem could get fix from that angle?

I have not, every time it's happened was when I was least prepared for it, though I believe this might just be a memory issue within GTK itself...
I've had Thunar crash on me for similar reasons, similar circumstances, and I know that's also built similarly

so I'd like to test it out just by running more daemons using less resources individually
at least if any of those crash, it shouldn't completely derail my workflow when I'm least prepared for it
ultimately not distracting me so much from collecting such logs
(or at least seeing when it happens in a terminal)


also while I'm at it, one last minor thing I'd like to suggest (to those involved), I don't feel this is big enough for it's own issue...
but I'd like to just point out the Devices category in the Places panel is rather broken in recent versions compared to 4.4.2 :)
perhaps a focus toward libudev and `/proc/mounts` would be more of interest over whatever GTK does to gather volume data? ;)

thanks to all who work with me on this :D

**Comment by Aleksanaa** [comment]
> update looks like the problem has to do with a dbus check in:
> 
>     * `libnemo-private/nemo-dbus-manager.c` for `"/org/Nemo"`
> 
>     * `src/nemo-freedesktop-dbus.c` for `"/org/freedesktop/FileManager1"`

Any suggestions on patching this file?
